### PR TITLE
"Last Update" on project card is not correct

### PR DESCRIPTION
**Pull request by RamRamez**
https://github.com/Giveth/giveth-dapps-v2/issues/4204

https://github.com/Giveth/impact-graph/issues/1583

## Summary by CodeRabbit

- **New Features**
  - Added `latestUpdateCreationDate` to project tracking, providing users with the latest update creation date for each project.
  - Introduced `useDonationBox` and `relevantDonationTxHash` columns to the donation and draft donation tables, enhancing donation tracking.

- **Updates**
  - Improved project update statistics calculation for more accurate reporting.
  - Enhanced environment variable handling in scripts for better development workflow.

- **Bug Fixes**
  - Ensured new columns are conditionally added in migrations to avoid conflicts with existing columns.

- **Testing**
  - Updated test cases to reflect changes in project update statistics and new properties in project and donation entities.

### CODE REVIEW

File: package.json
Comment (mohammadranjbarz) [inline package.json:1]: Please undo the changes on run test commands, it would not work on the `OS X`

**Review by mohammadranjbarz — CHANGES_REQUESTED** [review]
Thanks @RamRamez  LGTM, just I comemnted a feedback on `package.json`

File: package.json
Comment (RamRamez) [reply package.json:1]: Thanks @mohammadranjbarz.
Done!

**Review by mohammadranjbarz — APPROVED** [review]
Thanks so much

**Pull request merged**